include/baresip.h

```diff
@@ -1316,6 +1316,8 @@ struct config_audio *audio_config(struct audio *au);
  */
 
 struct video;
+struct vtx;
+struct vrx;
```

Comment:
these types are not supposed to be exposed outside video.c

it is supposed to be a hidden implementation specific detail.

Reply:
you are not supposed to call this method directly.

It is an implementation of the "vidcodec" interface.

Reply:
`av1_encode_update` was given as an example. Based on his code, I made a proprietary video source